Tasks/VSMobileCenterUpload/task.json

```diff
@@ -0,0 +1,149 @@
+{
+    "id": "B832BEC5-8C27-4FEF-9FB8-6BEC8524AD8A",
+    "name": "VSMobileCenterUpload",
+    "friendlyName": "Visual Studio Mobile Center Upload",
+    "description": "Upload mobile app packages to Visual Stuidio Mobile Center",
+    "helpMarkDown": "Upload mobile app packages to Visual Stuidio Mobile Center",
```

Comment:
helpMarkDown is generally a link to the documentation. Since it is a built in task, it makes sense to add docs and provide a link to that.